src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -772,12 +882,15 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             string explanation = default,
             ParseMode explanationParseMode = default,
             int? openPeriod = default,
-            DateTime? closeDate = default
+            DateTime? closeDate = default,
+            bool allowSendingWithoutReply = default,
+            IEnumerable<MessageEntity> explanationCaptionEntities = default
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.